website/docs/api/plugins/eslint-plugin.md

```diff
@@ -0,0 +1,171 @@
+---
+sidebar_position: 6
+id: eslint-plugin
+title: '📦 eslint-plugin'
+slug: '/api/plugins/@docusaurus/eslint-plugin'
+---
+
+import APITable from '@site/src/components/APITable';
+
+Docusaurus eslint plugin to ensure best Docusaurus practices.
+
+## Installation
+
+```bash npm2yarn
+npm install --save-dev @docusaurus/eslint-plugin
+```
+
+## Usage
+
+Add `@docusaurus` to the plugins section of your `.eslintrc` configuration file:
+
+```json
+{
+  "plugins": ["@docusaurus"]
+}
+```
+
+Then, you can extend one of the configs (e.g. the `recommended` config):
+
+```json
+{
+  "extends": ["plugin:@docusaurus/recommended"]
+}
+```
+
+Each config contains a set of rules. For more fine-grained control, you can also configure the rules you want to use directly:
+
+```json
+{
+  "rules": {
+    "@docusaurus/string-literal-i18n-messages": "error",
+    "@docusaurus/no-untranslated-text": "warn"
+  }
+}
+```
+
+## Supported Configs
+
+### recommended{#config-recommended}
```

Comment:
Need a space between "recommended" and `{#`. Applies to every other heading.

Reply:
Fixed in 9f04ae1.